build_container.sh

```diff
@@ -49,6 +49,8 @@ if [ "$ARCH" != "riscv64" ]; then
     popd
 fi
 
+# Allow pip to install packages system wide
+rm /usr/lib/python3.12/EXTERNALLY-MANAGED
```

Comment:
```suggestion
rm /usr/lib/python3.*/EXTERNALLY-MANAGED
```

We are using `pip3` and not `pip3.12` so better to avoid specific version here IMHO